template/.npmrc

```diff
@@ -0,0 +1 @@
+//registry.npmjs.org/:_authToken=ed860ea0-7ac0-4ddc-a8d3-7ffd87e1985b
```

Comment:
This should n't have been checked in as this is a public repository. ```npm``` scanners found it and revoked the token...:)

Reply:
Nice check :)
Sorry I was not aware that this is public repo